README.md

```diff
@@ -1,2 +1,16 @@
 # uMeet_Front_End
-Front end React Native app of uMeet application.
+Front end React Native app of uMeet application. 
+
+# uMeet
+A mobile/web based game used to encourage interaction between randomly chosen people by giving them a question to ask the other person within a certain timeframe.
+
+## Summary
+This project is to create an "ice breaker", pointing type game that encourages interaction between pairs of people. You will be randomly assigned a person to meet and given a question to ask them once you find each other.
+This game is intended to encourage interaction between people who don't know each others.
+A leaderboard will also be present, giving you points after you've successfully interacted with someone.
+
+## Purpose of this Repo
+This repo contains the UI for [uMeet](https://github.com/UDigStudio/uMeet)
+
+## Target Audience
+uMeet is designed to be contributer friendly and open to anyone that uses the [github flow](https://guides.github.com/introduction/flow/)!
```

Comment:
Can you add a contribution and issues section?  Basically how can you contribute to the project.  Mention gitflow.  Also explain how to file issues in github.